src/consensus/aft/raft.h

```diff
@@ -1833,18 +1833,28 @@ namespace aft
       timeout_elapsed = std::chrono::milliseconds(distrib(rand));
     }
 
+    void reset_votes_for_me()
+    {
+      for (auto const& conf : configurations)
```

Comment:
```suggestion
      votes_for_me.clear();
      for (auto const& conf : configurations)
```

Else we're also looking for quorum in extremely old/dead configurations?

Reply:
Thank you!